### DISCUSSION TITLE
Can't enable motor

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#161] [Q&A] [answered]
I am probably missing something simple, as usual, but I can't seem to make the motor activate. I am trying to do it the way it is done in example, with windmill that has a hinge constraint, but in my case it simply hangs under gravity.

https://github.com/jrouwe/JoltPhysics.js/assets/5677782/848d0e6c-5cc6-46a2-a450-46db4fcc83db

I do
```
joint.SetMotorState(Jolt.EMotorState_Velocity);
joint.SetTargetAngularVelocity(-10);
```
Any ideas how I could debug it? Not sure what I missed.

**Comment by jrouwe** [comment]
Hard to say what may be going wrong without the surrounding code.

I would start by creating a debug build for JoltPhysics.js and looking at the console to see if any asserts trigger.

**Comment by LeXXik** [comment, marked as answer]
It was a user error. I was using `Jolt.wrapPointer()` instead of `Jolt.castObject()` to get the hinge constraint.
